src/rebuild/srv.c

```diff
@@ -1226,21 +1227,21 @@ rebuild_task_complete_schedule(struct rebuild_task *task, struct ds_pool *pool,
 			return rc;
 		}
 
-		/* Schedule reclaim to clean up current op */
+		/* Schedule reclaim to clean up current op. Let's keep go ahead to retry even if
+		 * reclaim the current rebuilding fails.
```

Comment:
did we ever hit such reclaim failure in real test?
if reclaim failed and retry rebuild possibly hit the VOS same epoch partial overwrite failure.